examples/knative/powershell/kn-ps-vrni-databus/handler.ps1

```diff
@@ -0,0 +1,84 @@
+Import-Module CloudEvents.Sdk
+
+. ./vrni-functions.ps1
+
+Function Process-Init {
+   [CmdletBinding()]
+   param()
+   Write-Host "$(Get-Date) - Processing Init`n"
+
+   Write-Host "$(Get-Date) - Init Processing Completed`n"
+}
+
+Function Process-Shutdown {
+   [CmdletBinding()]
+   param()
+   Write-Host "$(Get-Date) - Processing Shutdown`n"
+
+   Write-Host "$(Get-Date) - Shutdown Processing Completed`n"
+}
+
+Function Process-Handler {
+   [CmdletBinding()]
+   param(
+      [Parameter(Position = 0, Mandatory = $true)]$Source,
+      [Parameter(Position = 1, Mandatory = $true)]$Body
+   )
+
+   # Retrieve webhook sink URL
+   $webhookSinkUrl = ${env:WEBHOOK_SINK_URL}
```

Comment:
nit: add a check if this var is not set and throw instead of continuing and enforcing a hardcoded default.

Reply:
Added a check if the ENV var exists, but I'd like to keep the default. Otherwise, the user would be forced to add the webhook_config.json, even though that's unnecessary for a single VEBA appliance deployment.